index.bs

```diff
@@ -2838,21 +2827,22 @@ The {{IDBObjectStore/name}} attribute's setter must run these steps:
 </div>
 
 
-The <dfn attribute for=IDBObjectStore>keyPath</dfn> attribute's getter
-must return [=/this=]'s [=object-store-handle/object store=]'s [=object-store/key path=], or
+The <dfn attribute for=IDBObjectStore>keyPath</dfn> getter steps are to
+return [=/this=]'s [=object-store-handle/object store=]'s [=object-store/key path=], or
 null if none. The [=/key path=] is converted as a {{DOMString}} (if a
 string) or a [=sequence&lt;DOMString&gt;=] (if a list of strings), per
```

Comment:
Preexisting issue: this could be nicer typographically if you used the awkward `<code><a interface>sequence</a>&lt;<a interface>DOMString</a>&gt;</code>`

Reply:
Done.